src/commands/xinfo-stream.json

```diff
@@ -7,6 +7,10 @@
         "arity": -3,
         "container": "XINFO",
         "history": [
+            [
+                "6.2.0",
```

Comment:
are you sure?
i see it in 6.0.0 too

Reply:
Makes sense. It was merged by antirez in 2020, which must have been before 6.0.0.

GitHub lists 1e2aee39192c8839170743d324c799860dc38c8c as included in a number of tags, but I guess the list of tags is truncated here...

Reply:
You are right! also available in 6.0 by 6c0bc608a19a5071fca7f755fccd4ed67961919